indico/modules/events/surveys/clone_test.py

```diff
@@ -0,0 +1,30 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.db.sqlalchemy.util.models import get_simple_column_attrs
+from indico.modules.events.surveys.clone import EventSurveyCloner
+from indico.modules.events.surveys.models.items import SurveyItem, SurveyQuestion, SurveySection
+from indico.modules.events.surveys.models.surveys import Survey
+
+
+def test_survey_clone(db, create_event, dummy_event):
+    survey = Survey(title='test')
+    section = SurveySection(title='test', display_as_section=True)
+    survey.items.append(section)
+    SurveyQuestion(title='question', field_type='text', is_required=True, parent=section)
```

Comment:
Maybe we could add a `SurveyText` item as well?

Reply:
I was actually thinking about parametrizing, but it's not that easy with parents. The latest test should ensure more correctness.